src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/DataGridViewComboBoxCellAccessibleObjectTests.cs

```diff
@@ -3,21 +3,22 @@
 // See the LICENSE file in the project root for more information.
 
 using Xunit;
+using static Interop;
 
 namespace System.Windows.Forms.Tests.AccessibleObjects
 {
     public class DataGridViewComboBoxCellAccessibleObjectTests
     {
         [Theory]
-        [InlineData(NativeMethods.UIA_IsExpandCollapsePatternAvailablePropertyId, true)]
+        [InlineData((int)UiaCore.UIA.IsExpandCollapsePatternAvailablePropertyId, true)]
         public void GetPropertyValue_Returns_Correct_Value(int propertyID, object expectedPropertyValue)
```

Comment:
You can change the type of `propertyID` to avoid boxing/unboxing.
```suggestion
         public void GetPropertyValue_Returns_Correct_Value(UiaCore.UIA propertyID, object expectedPropertyValue)
```

Reply:
Cannot, because Interop is an internal class to match conventions and the test method needs to have public parameters

Reply:
Okay, thanks for the explanation.